### ISSUE TITLE
BUG: Music not loading

### ISSUE THREAD

**Issue by XxLilJeepXx** [Musish/Musish#200]
Music not loading

**To Reproduce**
1. Sign in
2. Click on 'My Library'
3. Wait on loading screen

**Expected behavior**
Music from iTunes to load up on the recently added

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [MacBook Air]
 - Browser [Chrome]
 - Version [71.0.3578.98]
![screen shot 2019-01-19 at 2 16 26 am](https://user-images.githubusercontent.com/46838012/51424193-3df1ad00-1b90-11e9-97ba-dd42466d7385.png)

**Comment by codymikol** [comment]
This could potentially be a duplicate of https://github.com/Musish/Musish/issues/192

Do you have an apple music subscription?

**Comment by codymikol** [comment]
So you don't currently have an active subscription?

**Comment by BrychanOdlum** [comment]
Musish requires an active Apple Music subscription, this is by design of Apple's APIs. Closing issue.
